docs/app/routes/docs/guides/react-three-fiber.mdx

```diff
@@ -0,0 +1,401 @@
+    })
+  }
+
+  const handlePointerLeave = () => {
+    api.start({
+      scale: 1,
+    })
+  }
+
+  const handleWindowPointerOver = useCallback(() => {
+    isOver.current = true
+  }, [])
+
+  const handleWindowPointerOut = useCallback(() => {
+    isOver.current = false
+
+    api.start({
+      position: [0, 0],
+    })
+  }, [])
+
+  const handlePointerMove = useCallback(
+    e => {
+      if (isOver.current) {
+        const x = (e.offsetX / width) * 2 - 1
+        const y = (e.offsetY / height) * -2 + 1
+
+        api.start({
+          position: [x * 5, y * 2],
+        })
+      }
+    },
+    [api, width, height]
+  )
+
+  useEffect(() => {
+    window.addEventListener('pointerover', handleWindowPointerOver)
+    window.addEventListener('pointerout', handleWindowPointerOut)
+    window.addEventListener('pointermove', handlePointerMove)
+
+    return () => {
+      window.removeEventListener('pointerover', handleWindowPointerOver)
+      window.removeEventListener('pointerout', handleWindowPointerOut)
+      window.removeEventListener('pointermove', handlePointerMove)
+    }
+  }, [handleWindowPointerOver, handleWindowPointerOut, handlePointerMove])
+
+  return (
+    <animated.mesh
+      onPointerEnter={handlePointerEnter}
+      onPointerLeave={handlePointerLeave}
+      onClick={handleClick()}
+      scale={springs.scale}
+      position={springs.position.to((x, y) => [x, y, 0])}>
+      <sphereGeometry args={[1.5, 64, 32]} />
+      <AnimatedMeshDistortMaterial
+        speed={5}
+        distort={0.5}
+        color={springs.color}
+      />
+    </animated.mesh>
+  )
+})
+
+export default function MyComponent() {
+  const blobApi = useRef(null)
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (blobApi.current) {
+        const { x, y } = blobApi.current.getCurrentPosition()
+        console.log('the blob is at position', { x, y })
+      }
+    }, 2000)
+
+    return () => clearInterval(interval)
+  }, [])
+
+  return (
+    <Canvas>
+      <ambientLight intensity={0.8} />
+      <pointLight intensity={1} position={[0, 6, 0]} />
+      <MyScene ref={blobApi} />
+    </Canvas>
+  )
+}
+```
+
+## Troubleshooting
+
+### Experiencing Jank?
+
+Whilst jank in `react-three-fiber` cannot be purely blamed on `react-spring` you
+might find toward the end of an animation that there's a subtle jump,
+which is visible in [this demo](https://y18m1.csb.app/). It's not pretty, is it?
+
+Whilst by default it would be nice to have this issue resolved without you having
+to interact and this is something we'll consider for the next `breaking change`
+in the meantime what you can use is the [`precision` config prop](/common/configs#configs) to avoid this.
+
+By setting the prop to a value like `0.0001` you can notice there is no jump towards 0.
+This is because the precision prop is used to figure out how close the animated value
+can get to the end goal before we consider the animated value to be equal to the end goal.
```

Comment:
You'll want to give a code example or make this interactive for those referring to this guide for the troubleshooting section.